tests/python/contrib/test_ethosu/test_codegen.py

```diff
@@ -987,5 +990,36 @@ def split_func(x):
     _compare_tvm_with_tflite(split_func, [ifm_shape], accel_type)
 
 
+@pytest.mark.parametrize("accel_type", ACCEL_TYPES)
+@pytest.mark.parametrize(
+    "ifm_shape,ifm_scale,ifm_zp,ofm_scale,ofm_zp",
+    [
+        [(1, 8, 8, 3), 1.0, 0, 1.0, 0],
+        [(1, 20, 30, 3), 1.345, 34, 0.32, -23],
+    ],
+)
+def test_ethosu_requantize(accel_type, ifm_shape, ifm_scale, ifm_zp, ofm_scale, ofm_zp):
+    dtype = "int8"
+    ifm_shape = [1, 8, 8, 3]
```

Comment:
Looks like that `ifm_shape` always overwrites the `ifm_shape` that is specified by the parameter...

Reply:
oops, thanks!